packages/plugin-data/src/api/data/soql/soqlQuery.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { QueryResult } from 'jsforce';
+import { Connection, Logger } from '@salesforce/core';
+import { Tooling } from '@salesforce/core/lib/connection';
+import { Field, retrieveColumns } from './queryFields';
+
+/**
+ * Type to define SoqlQuert results
```

Comment:
query
[thanks, https://github.com/streetsidesoftware/vscode-spell-checker]

Reply:
👍 , but I use WebStorm, and it caught the typo. I chose to ignore it ;)